src/test/java/org/broadinstitute/hellbender/utils/gcs/BucketUtilsTest.java

```diff
@@ -57,6 +58,19 @@ public void testIsFileURL(){
         Assert.assertFalse(BucketUtils.isFileUrl("gs://abucket"));
     }
 
+    @Test
+    public void testAppendPathToDir() {
+	Assert.assertEquals(BucketUtils.appendPathToDir("dir", "file"), "dir/file");
+	Assert.assertEquals(BucketUtils.appendPathToDir("dir/", "file"), "dir/file");
+	Assert.assertEquals(BucketUtils.appendPathToDir("dir", "/file"), "/file");
+	Assert.assertEquals(BucketUtils.appendPathToDir("dir/", "/file"), "/file");
+	Assert.assertEquals(BucketUtils.appendPathToDir("/path/to/dir", "anotherdir/file"), "/path/to/dir/anotherdir/file");
+
+	Assert.assertEquals(BucketUtils.appendPathToDir("file://dir", "file"), "file://dir/file");
+	Assert.assertEquals(BucketUtils.appendPathToDir("hdfs://namenode:9000/dir", "file"), "hdfs://namenode:9000/dir/file");
+	Assert.assertEquals(BucketUtils.appendPathToDir("gs://abucket/dir", "file"), "gs://abucket/dir/file");
+    }
+
```

Comment:
I think we need some tests covering the new functionality introduced by this PR (ie., GCS support). Let us know if you need help setting this up -- there are examples of tests that access data in GCS buckets in `GCSNioIntegrationTest`

Reply:
@droazen, yes, I need some help setting up tests for using hdfs and gs URIs with GenomicsDB. I am taking a look at `GCSNioIntegrationTest`, but still not sure on setting up a test bucket with folder, etc. on GCS and getting authentication setup via GOOGLE_APPLICATIONS_CREDENTIALS for GenomicsDB.